src/components/app/ServiceWorkerManager.js

```diff
@@ -102,7 +102,8 @@ class ServiceWorkerManager extends PureComponent<Props, State> {
     if (
       dataSource !== 'none' &&
       dataSource !== 'public' &&
-      dataSource !== 'from-url'
+      dataSource !== 'from-url' &&
+      dataSource !== 'compare'
```

Comment:
Ah yes, it's nice to be able to reload here since we won't lose any data.
(It took me a few seconds to remember why we don't prompt the reload button for the other dataSources. Since we are here, maybe it would be good to write a comment explaining the data loss on other sources?)

Reply:
Should this be changed to a `switch` with an `assertExhaustiveCheck`? This way when the data sources change, we ensure that this notification stays up to date.

Reply:
I wonder about this too but wanted a quick fix. But maybe I should invest the time to do it so that this will never regress :)